Pod/Classes/ATTutorialController.m

```diff
@@ -94,8 +104,8 @@ -(void)setUpMask{
 }
 
 -(UILabel *)labelForTutorialWithText:(NSString *)title{
-    UILabel *label = [[UILabel alloc]initWithFrame: self.hasLabelPosition ? self.labelPosition : CGRectMake(20.0, 300.0, 280.0, 100.0)];
-    [label setFont:[UIFont fontWithName:@"Helvetica Neue" size:18.0]];
+    UILabel *label = [[UILabel alloc]initWithFrame: self.hasLabelPosition ? self.labelPosition : CGRectMake(30, ((self.window.frame.size.height) - 150.0), self.window.frame.size.width - 60, 100.0)]; // Vitor - Fixed label position
```

Comment:
This font setting has nothing to do with the current PR. 
A thing to do would be to accept the font as a config of the component. I can take a look a that in the future.